### ISSUE TITLE
alicloud_kvstore_instance, throwing warnings with terraform v0.11.11, and alicloud provider 1.28.0

### ISSUE THREAD

**Issue by DevOps-Airwallex** [terraform-providers/terraform-provider-alicloud#690]
### Terraform Version
```
$ terraform -v
Terraform v0.11.11
```

### Affected Resource(s)
alicloud provider: `1.28.0`
Please list the resources as a list, for example:
- `alicloud_kv_store`

### Terraform Configuration Files
N/A

### Debug Output
```
Warning: provider.alicloud: "fc": [DEPRECATED] Field 'fc' has been deprecated from provider version 1.28.0. New field 'fc' which in nested endpoints instead.



Warning: provider.alicloud: "log_endpoint": [DEPRECATED] Field 'log_endpoint' has been deprecated from provider version 1.28.0. New field 'log' which in nested endpoints instead.



Warning: provider.alicloud: "mns_endpoint": [DEPRECATED] Field 'mns_endpoint' has been deprecated from provider version 1.28.0. New field 'mns' which in nested endpoints instead.
```
### Panic Output
N/A

### Expected Behavior
should not have `warning`

### Actual Behavior
throwing `warnings`
![image](https://user-images.githubusercontent.com/28879854/51461837-1ec66b80-1d9a-11e9-9b8e-39aa98ac3ad2.png)

### Steps to Reproduce
below `terraform` resource
```
provider "alicloud" {
  access_key = "xxxx"
  secret_key = "xxxxxx"
  region     = "cn-shenzhen"
}

resource "alicloud_kvstore_instance" "default" {
  instance_class = "redis.master.small.default"
  instance_name  = "myredis"
  password       = "Passw0rd"
  vswitch_id     = "vsw-<>"
}
```
```
terraform init
terraform apply
```

### Important Factoids
N/A

### References
N/A

**Comment by xiaozhu36** [comment]
HI @DevOps-Airwallex I am sorry for that. This issue has been fixed by the release 1.29.0. Please update and try again.

**Comment by xiaozhu36** [comment]
I will close it and if it still exists, please reopen it.
